web/pandas/pdeps/0010-required-pyarrow-dependency.md

```diff
@@ -0,0 +1,160 @@
+# PDEP-10: PyArrow as a required dependency for default string inference implementation
+
+- Created: 17 April 2023
+- Status: Under discussion
+- Discussion: [#52711](https://github.com/pandas-dev/pandas/pull/52711)
+              [#52509](https://github.com/pandas-dev/pandas/issues/52509)
+- Author: [Matthew Roeschke](https://github.com/mroeschke)
+          [Patrick Hoefler](https://github.com/phofl)
+- Revision: 1
+
+## Abstract
+
+This PDEP proposes that:
+
+- PyArrow becomes a required runtime dependency starting with pandas 3.0
+- The minimum version of PyArrow supported starting with pandas 3.0 is version 7 of PyArrow.
+- When the minimum version of PyArrow is bumped, PyArrow will be bumped to the highest version that has
+  been released for at least 2 years.
+- The pandas 2.1 release notes will have a big warning that PyArrow will become a required dependency starting
+  with pandas 3.0.
+- Starting in pandas 2.2, pandas raises a ``FutureWarning`` when PyArrow is not installed in the users
+  environment when pandas is imported. This will ensure that only one warning is raised and users can
+  easily silence it if necessary.
```

Comment:
I recall some discussion we had on this PDEP about having the warning point to a GitHub issue where we could collect feedback on this requirement.  If we agree on this concept, I think it should be mentioned here.

Reply:
Added

Reply:
Do you mind adding that the warning will point to the feedback issue?

Reply:
Apologies for the naive question, but I thought the purpose of this PDEP was to collect feedback on pyarrow as a required dependency? I understand visibility may be more if there's a link to a new issue in a warning in a future version of pandas, but to me, doing it that way (to collect feedback all over again) just seems like we're going to be having this discussion again in 6 months and will end up kicking kicking the can down the road.